config/initializers/raven.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```

Comment:
TODO: set [SENTRY_ENVIRONMENT](https://github.com/scientist-softserv/atla-hyku/blob/2e938ea267a1719fa6f01b7277a65f7ae3bc0904/ops/production-deploy.tmpl.yaml#L179C11-L179C29) variable instead of adding an initializer file.